paper/paper.md

```diff
@@ -7,7 +7,7 @@ tags:
   - cognitive neuroscience
   - computational neuroscience
   - TRF
-  
+
```

Comment:
comment for authors below: did you forget the ORCIDs for the other authors or are they not available?

Reply:
Thank you for the tip regarding line breaks, I reformatted the Markdown file. Regarding the other questions:
1. Edmund Lalor is the senior author
2. The 3rd committer was a user who fixed a minor bug, do you think we should name an explicit inclusion criterion like minimum lines of code?

Reply:
@OleBialas thanks for your answers!

Regarding point 1 I have a small FYI for you: Please see the JOSS authorship guidelines: https://joss.readthedocs.io/en/latest/submitting.html?highlight=authorship#authorship

In particular this part:

> Purely financial (such as being named on an award) and organizational (such as general supervision of a research group) contributions are not considered sufficient for co-authorship of JOSS submissions, but active project direction and other forms of non-code contributions are. 

If E.L. still qualifies for authorship, that's great. If he doesn't, please discuss this internally and inform me about the action you want to take!

Regarding point 2: It's of course entirely up to you whom to include in this publication. I personally believe it is advisable to give credit to contributors, in order to encourage them to stick with the project and come back for future contributions. Transparent guidelines can be a valuable asset in that regard :+1:

Reply:
He definitely does, as one of the authors of the original mTRF-toolbox he guided the development process.

I agree with you about giving credit  to contributors, I'm just wondering what's the appropriate mechanism to do this (I don't think they should be authors on the paper because readers should approach us, not external contributors, with questions). 

I saw that you are keeping a list of authors in your doc files: https://github.com/mne-tools/mne-bids/blob/main/doc/authors.rst
However, I couldn't find what you were doing with that file when building the docs - is it just sitting there?

Reply:
> He definitely does, as one of the authors of the original mTRF-toolbox he guided the development process.

I'm happy to hear it! I consider this point resolved, thank you.

> I don't think they should be authors on the paper because readers should approach us, not external contributors, with questions

I agree 👍 

>  However, I couldn't find what you were doing with that file when building the docs - is it just sitting there?

in mne-bids we acknowledge contributors in several ways:

1. they get added to the changelog with a short description of their contribution (this is what the authors.rst file is for: it's simply providing a link registry for the authors listed in the whats_new.rst file)
2. they get added to our CITATION.cff file, which is parsed by zenodo, associating the authors with a particular software release

I think that perhaps looking into a CITATION.cff file would be valuable for your repository. I furthermore recommend that you extend your release protocol to include GitHub releases (and corresponding git tags); and to link your repository with zenodo, such that each release is automatically archived. Please let me know if you need more information on this.